tests/testthat/test-basic.R

```diff
@@ -10,7 +10,7 @@ test_that("Methods with variable number of parameters with c# 'params' keyword",
 
 test_that("Vignette examples work", {
   assemblyPath <- system.file("extdata", "rSharp.Examples.dll", package = "rSharp")
-  expect_equal(loadAssembly(assemblyPath)[[1]], assemblyPath)
+  expect_true(loadAssembly(assemblyPath))

```

Comment:
I'm not sure now if you need to modify any documentation about this function but it will return `TRUE` when passing and an error string when failing